liberapay/constants.py

```diff
@@ -344,8 +344,8 @@ def __missing__(self, currency):
 )
 
 PRIVACY_FIELDS = OrderedDict([
-    ('hide_giving', (_("Do not publish the amounts of money I send."), False)),
-    ('hide_receiving', (_("Do not publish the amounts of money I receive."), False)),
+    ('hide_giving', (_("Do not publish the amount of money I send."), False)),
```

Comment:
https://english.stackexchange.com/questions/254339/when-to-use-amount-vs-amounts

Reply:
Yes. This is also wrong in "Source code of the recurrent donations platform Liberapay"